### ISSUE TITLE
Segmeth identification of 5mC at CG motif

### ISSUE THREAD

**Issue by JuandeBarba** [adamewing/methylartist#74]
Hi all,

first, I wanted to thank you for the nice software. I was just having a naive question regarding how segmeth extracts the data from BAM files. I have some Nanopore based methylation data (R10 Flowcells), basecalled with Dorado for 5mC at CG, aligned and indexed. I want to extract the methylation status from an specific region in 1kb windows.

When I do segmeth, motif-- CG, I obtain an tsv file with meth_calls, unmeth_calls, no_calls and methfraction per 1000 bp window. 
How is it defined a meth_call? Does it require all CG present in the window to be methylated or only some of them? What is the threshold (from MM-ML tags) to consider a C to be methylated?

Thanks a lot for the time!

Juande

**Comment by adamewing** [comment]
Hi Juande,

`meth_calls` (and equivalently for `unmeth_calls` and `no_calls`) counts up the number of CGs with a methylation score above 0.8 (i.e. n/255 > 0.8 where n is the ML value). Note this isn't the number of methylated CG sites in the genome, it's the number of methylated CG sites in the collection of reads covering the interval. Does that help?

--Adam

**Comment by JuandeBarba** [comment]
Hi Adam,

thanks a lot! It helps greatly. Then, if I understood correctly, methfraction is the proportion of methylated CG over the total of CG detected in our reads.

Thanks again for the quick reply!

Juande

**Comment by adamewing** [comment]
Yes where total CG is methylated + unmethylated sites (ambiguous sites i.e. those with methylation score/255 between 0.2 and 0.8 are ignored)
